docs/skinning.md

```diff
@@ -0,0 +1,116 @@
+# Skinning Your uPortal-home #
+
+_Prerequisites:_
+
+_A working uPortal-app-framework application._
+
+_A code editor._
+
+For a more in-depth discussion of theming, see https://github.com/UW-Madison-DoIT/uw-frame/blob/master/docs/theming.md
+
+1. Locate and open the frame-config.js file in your uw-frame-components/js/ directory.
+
+
+2. Increment the themeVersion in line 9. 
+
+3.  Add the following json to the themes array:
+
+
+```json
+{
+  'name': '',
+  'crest': 'img/uw-madison.png',
+  'title': '',
+  'subtitle': null,
+  'ariaLabelTitle': '',
+  'crestalt': '',
+  'group': 'Everyone',
+  'mascotImg': 'img/robot-taco.gif',
+  'footerLinks': [
+    {
+      'url': 'https://www.merriam-webster.com/dictionary/help',
+      'target': '_blank',
+      'title': 'Help',
+    },
+  ],
+  'materialTheme': {
+    'primary': 'red',
+    'accent': 'blue',
+    'warn': 'orange',
+  },
+},
+
+```
+   Fill in the blank values to reflect your school. 
+     * name: The name value you choose will become a filename in subsequent steps. The convention is all lower-case with no spaces.  
+     * title: The display name of your frame.  
+     * ariaLabelTitle: The aria label of your title.   
+     * crestAlt: Used to place a small bit of text (e.g. "Beta") next to your school's crest.  
+     
+
+4. In the \/uw-frame-components/css/themes directory, add a <theme-name>.less file. The theme name corresponds to the name attribute you assigned in the step above. 
+
+   Add the following imports:
+
+```sass
+@import "../angular.less"; // note: order is important here!
+@import "common-variables.less";
+@import "<your-schools-name>-variables.less";
+
+```
+
+
+
+5. Also in the /uw-frame-components/css/themes directory, add a <theme-name>-variables.less file. Copy and paste the following code into the file. These are the colors for UW-Madison. Feel free to use a color picker, such as http://www.designskilz.com/colors to generate your own colors, and experiment with the palette.
```

Comment:
skilz with a z! :sunglasses:

Any reason for the name drop of that site in particular?